sonic-xcvrd/xcvrd/xcvrd_utilities/port_mapping.py

```diff
@@ -134,6 +134,12 @@ def subscribe_port_update_event(namespaces, logger):
                                         port_tbl, list(d.values())[0], namespace))
     return sel, asic_context
 
+def apply_filter_to_fvp(filter, fvp):
+    if filter is not None:
+        for key in fvp.copy().keys():
```

Comment:
why copy()?

Reply:
Using a copy of the fvp while iterating will prevent running into issues caused by dynamically changing fvp since we will be deleting unwanted entries from fvp as we iterate.